src/agent/agentPB.ts

```diff
@@ -0,0 +1,4 @@
+export * as common from '../proto/js/domains/Common_pb';
```

Comment:
This seems strange, why export things from proto js if you're inside `agent`. I don't think this should be done... Again we want to avoid too much recursive imports/exports, it complicates the structure.

Reply:
It's just to make it clearer in the code when using the messages. I wanted to replicate the 'name spacing' so in the proto if the message was defined in vaults EG `vaults.Vault` then we'd refer to it in the code as `messages.vaults.Vault`. 

It also allows us to import the messages from one place.

Reply:
Yea that's fine, but I wouldn't export this from `src/agent/index.ts`. The `src/agent/index.ts` and `src/client/index.ts` should only export what's in the local directory.

And yes it should be imported from JS directly. Just don't export it from `src/agent/index.ts`. These things are not in the `agent` directory. They are generate JS files from `src/proto/js`.

Reply:
Fixed.